packages/deployer/src/infrastructure/index.ts

```diff
@@ -99,7 +99,7 @@ async function deploy(
   await runCommand(command, options);
 
   const httpGatewayApiKeyVar = httpGatewayApiKey ? `-var='api_key=${httpGatewayApiKey}'` : '';
-  command = `terraform apply -var="aws_region=${region}" -var="airnode_id_short=${airnodeAddressShort}" -var="stage=${stage}" -var="configuration_file=${path.resolve(
+  command = `terraform apply -var="aws_region=${region}" -var="airnode_address_short=${airnodeAddressShort}" -var="stage=${stage}" -var="configuration_file=${path.resolve(
```

Comment:
😮 it seems we missed this while merging `beta-protocol-revision`

Reply:
Yeah, but this might be just this file was not touched in the rebase (maybe it was renamed or whatever) and I didn't have merge conflict with it. 

Anyway, these were the only changes that I needed to do to make airnode starter work - so I am still confident about the rebase :)